src/tools/crawling_tools.py

```
@@ -16,6 +16,9 @@
 from crawl4ai import CrawlerRunConfig, CacheMode
 from src.core.timeout_utils import with_crawler_timeout, with_batch_timeout, with_timeout, TimeoutConfig
 from src.core.validation import validate_crawl_dir_params
+from src.utils.progress_utils import ProgressReporter
+from src.utils.error_handling import MCPErrorHandler, ErrorCategory, ErrorSeverity, create_success_response
+from src.utils.logging_utils import log_operation, MCPLogger, OperationMonitor
```

Ruff (0.12.2)

`src.utils.logging_utils.MCPLogger` imported but unused

Remove unused import: `src.utils.logging_utils.MCPLogger`

(F401)